### PR TITLE
Update containerd to v1.2.6

### PR DESCRIPTION
**Pull request by rn**
Also bump runc to the recommended version.

weasel-riding-woodpecker

### CODE REVIEW

File: tools/alpine/Dockerfile [hunk 1]
```diff
@@ -56,7 +56,7 @@ RUN go get -u github.com/LK4D4/vndr
 # Update `FROM` in `pkg/containerd/Dockerfile`, `pkg/init/Dockerfile` and
 # `test/pkg/containerd/Dockerfile` when changing this.
 ENV CONTAINERD_REPO=https://github.com/containerd/containerd.git
-ENV CONTAINERD_COMMIT=v1.2.5
+ENV CONTAINERD_COMMIT=v1.2.6
```
Comment (thaJeztah) [inline tools/alpine/Dockerfile]: Does bumping this version also bump runc to the same version as is used in the containerd's vendor.conf? (just double checking)

Comment (rn) [reply tools/alpine/Dockerfile]: no, but the subsequent commit (https://github.com/linuxkit/linuxkit/pull/3327/commits/fd67969697c3be7b8cea2268de235f792e7a5f2a) bumps `linuxkit/runc` to the recommended version

Comment (thaJeztah) [reply tools/alpine/Dockerfile]: Ah, thanks!

Comment (thaJeztah) [reply tools/alpine/Dockerfile]: LOL, shouldn't be looking at large diffs from my phone

**Pull request merged**